testing-standard.md

```diff
@@ -1,20 +1,21 @@
 # HX Testing Standards
 
 # 1.0 Unit Tests
+The aim is for our unit test output to describe the functionality of our code. As long as our tests are readable and clearly structured, we should focus our time on ensuring we have high quality tests. Being able to use the test output as a description for a piece of work, where appropriate, ensures our code is unambiguous and that are tests cover expected functionality. We _should_ be able to use our test output as documentation to our code.
```

Comment:
How about `we should focus our time on ensuring we have high quality tests which are readable and clearly structured`?